### ISSUE TITLE
Allow denoliver to be used programmatically

### ISSUE THREAD

**Issue by mdubourg001** [joakimunge/denoliver#9]
Hey, great work with Denoliver ! Is there any future plan to allow a full programmatic usage of the lib ?

Atm, the function exported from `mod.ts` is usable but does not enable the use of the options usable from cli (`--port`, `--silent`...). 

Example of usage could be something like:
```typescript
import serve from "https://deno.land/x/denoliver/mod.ts";

serve({ p: 3000, s: true, n: false, root: './dist' })
```

**Comment by joakimunge** [comment]
@mdubourg001 Hey, and thank you!

Yes! I have it in the pipe for V2 to support programmatic use. If I have time it will be ready and released this weekend :)

Thanks for giving denoliver a try!

**Comment by joakimunge** [comment]
@mdubourg001 Hi, just wanted to let you know that I finally released v2.0.0, which includes this feature! Hope you like it
